migrations2/2.1_salable.js

```diff
@@ -0,0 +1,6 @@
+const SalableDeployer = require('token-sale-contracts/migrations/deployers/Salable');
```

Comment:
File not needed

Reply:
we deploy external Contracts like VerifierRegistry, HumanStandardToken, etc. during the tests. truffle needs to generate ABI files so we can work with the contracts.